markdown-mode.el

```diff
@@ -7770,7 +7791,13 @@ Standalone XHTML output is identified by an occurrence of
     (insert markdown-xhtml-header-content))
   (insert "\n</head>\n\n"
           "<body>\n\n")
+  (when (> (length markdown-xhtml-body-preamble) 0)
+    (insert markdown-xhtml-body-preamble))
+  (insert "\n")
```

Comment:
It might be better to put this newline inside the `when` block, so that existing output for those who haven't customized this variable will be exactly the same.